templates/admin-page.php

```diff
@@ -43,10 +43,10 @@
 					<tr>
 						<td>
 							<fieldset>
-								<legend class="screen-reader-text"><?php esc_html_e( $category ); ?></legend>
+								<legend class="screen-reader-text"><?php echo esc_html( $category ); ?></legend>
```

Comment:
`esc_html_e` seems appropriate in both cases here. Is there a reason why these are being changed?

Reply:
Agreed, this doesn't look wrong to me.

Reply:
When using `esc_html_e`, the error `WordPress.WP.I18n.NonSingularStringLiteralText` is shown by PHPCS, which is related to how WordPress handles translation functions for plural and singular text strings. The problem arises from using the `esc_html_e()` function with a variable as the text parameter.

Reply:
Thanks for the explanation, @mukeshpanchal27. Of course you can't translate a variable. This change looks correct 👍🏻